scholia/app/templates/about.html

```diff
@@ -22,6 +24,18 @@ <h2>FAQ</h2>
 <h2>Presentations</h2>
```

Comment:
There are quite a few more, and we should try to gather an overview somewhere, probably here or on wiki.
Some starting points (all with false positives):
- [YouTube](https://www.youtube.com/results?search_query=scholia&sp=CAI%253D)
- [Zenodo](https://zenodo.org/search?page=1&size=20&q=scholia&sort=mostrecent)
- [WikiProject University of Virginia](https://en.wikipedia.org/wiki/Wikipedia:WikiProject_University_of_Virginia)